centreon/packages/ui/src/Form/Inputs/Checkbox.tsx

```diff
@@ -0,0 +1,96 @@
+import { ChangeEvent } from 'react';
+
+import { makeStyles } from 'tss-react/mui';
+import { FormikValues, useFormikContext } from 'formik';
+import { path, split } from 'ramda';
+
+import {
+  FormControlLabel,
+  Checkbox as MuiCheckbox,
+  FormGroup,
+  Box
+} from '@mui/material';
+
+import { useMemoComponent } from '../..';
+
+import { InputPropsWithoutGroup } from './models';
+
+const useStyles = makeStyles()((theme) => ({
+  checkbox: { padding: theme.spacing(0.5) },
+  container: {
+    alignItems: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    width: 'fit-content'
+  },
+  icon: {
+    fontSize: theme.spacing(12)
+  },
+  text: {
+    fontSize: theme.spacing(1.5)
+  }
+}));
+
+const MultiCheckbox = ({
+  change,
+  checkbox,
+  fieldName
+}: InputPropsWithoutGroup): JSX.Element => {
+  const { classes } = useStyles();
+
+  const { values, setFieldValue } = useFormikContext<FormikValues>();
```

Comment:
This component must not depend on Formik to work. I would add two props that handle a value and a value setter